sklearn/cluster/k_means_.py

```diff
@@ -98,7 +97,7 @@ def _k_init(X, n_clusters, x_squared_norms, random_state, n_local_trials=None):
     # Initialize list of closest distances and calculate current potential
     closest_dist_sq = euclidean_distances(
         centers[0, np.newaxis], X, Y_norm_squared=x_squared_norms,
-        squared=True)
```

Comment:
There is a check_arrays in the fit method of MiniBatchKmeans, but there doesn't seem to be one that happens before this one in regular KMeans.